frontend/views/containers/Overview.vue

```diff
@@ -1,139 +1,284 @@
-
-      label.label
+  .c-chart
+    .c-chart-legends
+      .c-legend
+        span.square.has-background-warning-solid
+        i18n Total needed
+        b {{ totalNeeded }}
+
+      .c-legend
+        span.square.has-background-primary-solid
         i18n Total pledged
-        .c-total-pledge $800
+        b {{ totalPledge }}
 
-      label.label
+      .c-legend(v-if='positiveBalance')
+        span.square.has-background-success-solid
         i18n Surplus
-        .c-surplus $50
+        b {{ surplus }}
+
+      .c-legend(v-else)
+        span.square.has-background-danger-solid
+        i18n Needed
+        b {{ surplus }}
+
+    .c-chart-wrapper
+      svg(
+        width='100%'
+        :viewBox='`0 0 ${ratioX} ${ratioY + verticalPadding}`'
+        preserveAspectRatio='xMidYMid meet'
+        aria-labelledby='title'
+        role='img'
+      )
+        // Move graph origin to the middle
+        g(:transform='`translate(0,${middle})`')
+          // Animate using scale from the middle
+          g.g-animate(:style='`transform: scale3d(1,${ ready ? 1 : 0 },1)`')
+            g.bars(
+              v-for='(member, index) in members.list'
+              :transform='positionX(index)'
+            )
+              // Total needed or total pledge bars
+              path(
+                :class='color(member, false)'
+                :d='roundedRect(member >= 0, 0, positionY(member), width, height(member), 3)'
+              )
 
-      i18n.help(tag='p') This amount will not be used
+              // Needs or surplus bars
+              path.g-animate-opacity.g-animate-delay(
+                v-if='hasSurplus(member)'
+                :class='color(member, true)'
+                :d='surplusRectangle(member)'
+                :style='`opacity: ${ ready ? 1 : 0 }`'
+              )
+
+          // Base with $0 on top of bars
+          g.line
+            line(x1='0' y1='0' :x2='ratioX' y2='0' stroke='#dbdbdb' stroke-width='1')
+            foreignObject(:x='ratioX - 25' :y='-labelPadding' width='25' height='18')
+              .tag.mincome {{ base }}
+
+          // Surplus line on top of bars
+          g.line(
+            :transform='`translate(0, ${-surplusPosition})`'
+          )
+            line.g-animate.g-animate-delay(
+              :style='`transform: scale3d(${ ready ? 1 : 0 },1,1)`'
+              x1='0'
+              y1='0'
+              :x2='ratioX'
+              y2='0'
+              stroke='#dbdbdb'
+              stroke-width='1'
+              stroke-dasharray='1'
+            )
+            // Income label
+            foreignObject(
+              :x='ratioX - mincome.length * 10'
+              :y='-labelPadding'
+              :width='mincome.length * 10'
+              height='18'
+              :transform='`translate(0,${surplusLabelPosition})`'
+            )
+              .tag.g-animate-opacity.g-animate-delay(
+                :style='`opacity: ${ ready ? 1 : 0 }`'
+              ) {{ mincome }}
```

Comment:
When loading the dashboard for the first time, the following warning appears in the console:

```
Unexpected value translate(0, NaN) parsing transform attribute. vue.esm.js:6790:7
Unexpected value translate(0,NaN) parsing transform attribute. vue.esm.js:6790:7
```